src/app/pipes/propertydisplay.ts

```diff
@@ -20,3 +20,24 @@ export class PropertyDisplayPipe implements PipeTransform {
     return rv;
   }
 }
+
+@Pipe({name: 'propertyValueDisplay'})
```

Comment:
There is a function doing the same thing in `storysidebar.component.ts` . Is it possible to replace that function with this pipe too?

Reply:
I believe I did. Can you not find it?

Reply:
ah, found it! sorry about that, i had to dig it up :p